assets/app/scripts/filters/resources.js

```diff
@@ -779,16 +779,8 @@ angular.module('openshiftConsole')
       }
     };
   })
-  .filter('humanizeKind', function () {
-    return function(kind) {
-      if (!kind) {
-        return kind;
-      }
-
-      // ReplicationController -> Replication Controller
-      // https://lodash.com/docs#startCase
-      return _.startCase(kind);
-    };
+  .filter('humanizeKind', function (startCaseFilter) {
```

Comment:
I'm not sure I see the benefit between Applies and Matches.  The match
terminology was chosen to be consistent with selectors which scopes align
to closest.  Can we stick with consistency in 3.2 between CLI and UI ?

On Thursday, March 17, 2016, Jessica Forrester <notifications@github.com>
wrote:

> In assets/app/scripts/filters/resources.js
> <https://github.com/openshift/origin/pull/8095#discussion_r56545814>:
>
> > @@ -779,16 +779,8 @@ angular.module('openshiftConsole')
> >        }
> >      };
> >    })
> > -  .filter('humanizeKind', function () {
> > -    return function(kind) {
> > -      if (!kind) {
> > -        return kind;
> > -      }
> > -
> > -      // ReplicationController -> Replication Controller
> > -      // https://lodash.com/docs#startCase
> > -      return _.startCase(kind);
> > -    };
> > +  .filter('humanizeKind', function (startCaseFilter) {
>
> ok we can leave it, doesnt hurt anything
> … <#-2373171795965911247_>
> On Thu, Mar 17, 2016 at 1:32 PM, Sam Padgett <notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote: In
> assets/app/scripts/filters/resources.js <#8095 (comment)
> <https://github.com/openshift/origin/pull/8095#discussion_r56544854>>: >
> @@ -779,16 +779,8 @@ angular.module('openshiftConsole') > } > }; > }) > -
> .filter('humanizeKind', function () { > - return function(kind) { > - if
> (!kind) { > - return kind; > - } > - > - // ReplicationController ->
> Replication Controller > - // https://lodash.com/docs#startCase > -
> return _.startCase(kind); > - }; > + .filter('humanizeKind', function
> (startCaseFilter) { Or change the case to lower case? We're inconsistent
> about that today. — You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <
> https://github.com/openshift/origin/pull/8095/files/e63659923bebc75526886378f48427f234695496#r56544854>
>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/8095/files/e63659923bebc75526886378f48427f234695496#r56545814>
>